internal/daysteps/daysteps.go

```diff
@@ -13,8 +19,55 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	slice := strings.Split(data, ",")
+	if len(slice) != 2 {
+		return 0, 0, fmt.Errorf("Размер данных не соответствует размеру пакета")
+	}
+
+	numberOfSteps, err := strconv.Atoi(slice[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("Ошибка преобразования")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```